### PR TITLE
[#5] [UI] [iOS] As a user, I can see the splash screen

### LINKED ISSUE #5: [UI] [iOS] As a user, I can see the splash screen
## Why

To make the first impression to the user, show the splash screen when opening the application.

## Acceptance criteria

Show the Splash screen.

## Design

- [Design for the splash screen](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=34%3A450&scaling=scale-down).

### PR DESCRIPTION
**Pull request by thiennguyen0196**
- Close #5 

## What happened 👀

Add background splash and logo white into assets
Create Splash View and integrate it in iOS App

## Insight 📝

N/A

## Proof Of Work 📹

https://github.com/nimblehq/ic-kmm-kayla-bruce/assets/25218255/81395e45-95f9-4b00-83f9-e7fe7ed3c32f

### CODE REVIEW

**Review by markgravity — APPROVED** [review]

File: ios/ios/SplashView.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//  ios
+//
+//  Created by Thiên Nguyễn on 8/10/23.
+//  Copyright © 2023 orgName. All rights reserved.
+//
+
+import SwiftUI
+
+let SHOW_LOGO_DELAY = 0.5
+let SHOW_LOGO_DURATION = 1.0
```
Comment (nmint8m) [inline ios/ios/SplashView.swift:12]: In Swift convention, we use lowerCamelCase for naming variables; these should be too. 

Besides, Ideally your global constants will be located within an `enum`, `extension`, or `struct` (just like `Fonts`).

If you still want to keep this approach, please consider setting it with the `private`  keyword.

```suggestion
private let showLogoDelay = 0.5
private let showLogoDuration = 1.0
```

File: ios/ios/SplashView.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+struct SplashView: View {
+
+    @State private var isLoaded = false
+
+    var body: some View {
+        ZStack {
+            Image("BackgroundSplash").resizable()
```
Comment (nmint8m) [inline ios/ios/SplashView.swift:21]: Please bring `.resizable()` to the new line.

```suggestion
            Image("BackgroundSplash")
            	.resizable()
```

File: ios/ios/SplashView.swift [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                .aspectRatio(contentMode: .fill)
+
+            Image("LogoWhite")
+                .aspectRatio(contentMode: .fill)
+                .frame(maxWidth: .infinity)
+                .opacity(
+                    isLoaded ? 1.0 : 0.0
+                )
```
Comment (nmint8m) [inline ios/ios/SplashView.swift:29]: I think we can write this in one line.

File: ios/ios/SplashView.swift [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        }
+        .ignoresSafeArea()
+        .animation(.easeIn(duration: SHOW_LOGO_DURATION).delay(SHOW_LOGO_DELAY), value: isLoaded)
```
Comment (nmint8m) [inline ios/ios/SplashView.swift:32]:
```suggestion
        .animation(
        	.easeIn(duration: showLogoDuration).delay(showLogoDelay), 
        	value: isLoaded
       	)
```

File: ios/ios/SplashView.swift [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        }
+    }
+}
+
+struct SplashView_Previews: PreviewProvider {
+
+    static var previews: some View {
+        SplashView()
+    }
+}
```
Comment (nmint8m) [inline ios/ios/SplashView.swift:44]: When you install the MacOS Beta with Xcode 15 beta, you can simple preview the view with Swift macro `#Preview`

```suggestion
#Preview {
    SplashView()
}
```

**Review by nmint8m — COMMENTED** [review]
Just some small comments. Rest LGTM.

**Review by nmint8m — APPROVED** [review]
Look good to me now 👍

**Pull request merged**